service/src/main/kotlin/app/cash/backfila/client/BackfilaClientServiceClientProvider.kt

```diff
@@ -24,6 +24,11 @@ class BackfilaDefaultEndpointConfigModule() : KAbstractModule() {
 }
 
 interface BackfilaClientServiceClientProvider {
+  fun headersSize(headers: List<HttpHeader>): Int {
+    return headers.joinToString("\r\n") { header -> "${header.name}: ${header.value}" }
+      .toByteArray()
+      .size
```

Comment:
I'm also not sure this belongs here? it is more a property of the header than the provider.